tests/kernel/mellanox_ofed.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Mellanox OFED package installation
+# This package is not ready for SLE15 yet, there is an available
+# package for SLE12-SP3 which is able to build the needed RPMs
+# for the current OS version.
+# Maintainer: Jose Lausuch <jalausuch@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-ssh';
+    my $ofed_version = 'MLNX_OFED_LINUX-4.3-1.0.1.0-sles12sp3-x86_64';
```

Comment:
I'm not sure here if we should hard-code this. Maybe allow overriding this with a Variable and use this string as default? In this case, you could avoid changing the code for new Versions?

Reply:
I can create 2 variables:
OFED_URL=http://content.mellanox.com/ofed/MLNX_OFED-4.3-1.0.1.0
OFED_FILE=MLNX_OFED_LINUX-4.3-1.0.1.0-sles12sp3-x86_64

do you agree or have a better suggestion?

Reply:
I agree. If you want to have it more simple, just use one variable containing the full URL to the file, this would be fine as well.

Reply:
Yes, I thought about that but then I need to parse in perl the url to get the filename and extension. Will do so then :)

Reply:
@frankenmichl you can ignore the Travis failure as it is not related to this commit.